Dockerfile

```diff
@@ -95,6 +95,8 @@ RUN ansible-galaxy collection install ./dist/networktocode*.tar.gz
 # Switch to the collection path for tests
 WORKDIR ${ANSIBLE_COLLECTIONS_PATH}/ansible_collections/networktocode/nautobot
 
+RUN ansible --version
```

Comment:
Do we need this change still?

Reply:
I would typically leave it in from a debugging perspective, even though it adds a layer. I will remove with the question ask.